src/components/schema/schema_item.test.tsx

```diff
@@ -0,0 +1,24 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+import React from 'react';
+import { render } from 'enzyme';
+import { requiredProps } from '../../test/required_props';
+
+import { OuiSchemaItem } from './schema_item';
+
+describe('OuiSchema', () => {
+  test('is rendered', () => {
```

Comment:
Obviously we'll eventually want more tests. Followup issue?

Reply:
Let me add a few. This component isnt set in stone yet so i'll hold back on adding a comprehensive test suite for now.